lib/src/algorithms/rsa.dart

```diff
@@ -64,8 +64,7 @@ class RSAKeyParser {
       return _parsePrivate(_pkcs8PrivateSequence(_parseSequence(rows)));
     }
 
-    // NOTE: Should we throw an exception?
-    return null;
+    throw ArgumentError('Unable to parse key, invalid format.');
```

Comment:
I left this note for further discussion.
Why do you think we should throw an exception?

Reply:
Because the methods taking the parsed key as an argument will most likely throw an exception if the key is null. Fail-fast would be better in this case.

Reply:
Do you agree that a [FormatException](https://api.dartlang.org/stable/2.2.0/dart-core/FormatException-class.html) is better is this case?

Reply:
Yes, absolutely. Will fix it right away.